src/OrleansPSUtils/Orleans.psd1

```diff
@@ -0,0 +1,24 @@
+﻿@{
+	RootModule = 'OrleansPSUtils.dll'
+	ModuleVersion = '1.2.3'
```

Comment:
Sorry I missed that earlier.

It could be nice to have this value synced with what is inside `Build\GlobalAssemblyInfo.cs`. I believe it could be done by writing a custom target in the csproj.

Same for the version in ReleaseNotes